### PR TITLE
Remove S2 prefix from docs storybook

### PR DESCRIPTION
**Pull request by yihuiliao**
Adds chromatic stories for all our S2 components and removes the S2 prefix so that our S2 stories don't fall under that heading.

We still need to improve our S2 chromatic story coverage since I mostly just copy and pasted from our docs storybook but figured we can get this in first and then work on that.

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

### CODE REVIEW

File: .chromatic/main.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ module.exports = {
   },
   stories: [
     '../packages/**/chromatic/**/*.stories.@(js|jsx|ts|tsx)',
-    '../packages/@react-spectrum/s2/stories/*.stories.@(js|jsx|mjs|ts|tsx)',
```
Comment (yihuiliao) [inline .chromatic/main.js:9]: removed this since we don't need to run chromatic on our s2 docs storybook anymore

**Comment by yihuiliao** [comment]
question about this line:
https://github.com/adobe/react-spectrum/blob/246225b096442a39f1f91b351f4ff88d9707ca2d/.chromatic-fc/main.js#L9

now that we've gotten rid of the S2 prefix, all the S1 and S2 story's get jumbled together so it's hard to tell which is which. Do we want to create a separate chromatic-fc for S2 or run it on our S2 chromatic storybook or leave it like this for now?

File: packages/@react-spectrum/s2/stories/ActionButton.stories.tsx [hunk 2]
```diff
@@ -20,18 +20,17 @@ import './unsafe.css';
...
   component: ActionButton,
   parameters: {
-    layout: 'centered'
-    // TODO: uncomment when baseline for new S2 chromatic stories is accepted since these are resused in the chromatic stories
-    // chromatic: {
-    //   disableSnapshot: true
-    // }
+    layout: 'centered',
+    chromatic: {
+      disableSnapshot: true
```
Comment (yihuiliao) [inline packages/@react-spectrum/s2/stories/ActionButton.stories.tsx:25]: do we still need this `disableSnapshot` if we don't plan to run chromatic on the normal docs stories?

Comment (devongovett) [reply packages/@react-spectrum/s2/stories/ActionButton.stories.tsx:25]: I don't think we should need it anymore

**Comment by snowystinger** [comment]
> question about this line:
> 
> https://github.com/adobe/react-spectrum/blob/246225b096442a39f1f91b351f4ff88d9707ca2d/.chromatic-fc/main.js#L9
> 
> now that we've gotten rid of the S2 prefix, all the S1 and S2 story's get jumbled together so it's hard to tell which is which. Do we want to create a separate chromatic-fc for S2 or run it on our S2 chromatic storybook or leave it like this for now?

I think we should change it to run against our chromatic stories instead?

**Comment by devongovett** [comment]
Can we separate the docs and these other examples for style macro? I'd like to just have all the docs at the top without needing to open a sub-folder.

image

Well should probably just be a real component anyway. It's actually quite a bit of styles for everyone to paste. And since these examples don't show the code, you'd need to find the story source in order to copy it so not sure how useful it is to show here.

**Comment by devongovett** [comment]
Oh, never mind, these stories are excluded from the docs anyway.

**Comment by reidbarber** [comment]
@devongovett the Well story was just added for us to verify of the codemod styles used to replace it.

**Review by devongovett — APPROVED** [review]

**Review by reidbarber — APPROVED** [review]

**Pull request merged**